runtime/doc/ui.txt

```diff
@@ -829,5 +831,10 @@ events, which the UI must handle.
 ["msg_history_clear"] ~
 	Clear the |:messages| history.
 
+							*ui-event-msg_intro*
+["msg_intro", lines] ~
```

Comment:
Instead of a special-purpose event would it make sense to add more fields to `msg_show`, such as (psuedo-names):  `width`, `height`, `x_offset`, `y_offset`. Then when `kind="intro"` UIs can choose to display it until user input.

Reply:
The problem with that is that it requires `ext_messages`, which also implicitly enables `ext_cmdline`, so then we are forced to handle both of those manually.

In Neovide, we have made the decision to keep our UI as true to the regular Neovim UI as possible, and externalizing those makes that much harder, since the layout no longer matches.

Reply:
The serialization format could use easily `msg_show[kind="intro"]` just to avoid the extra event, but it will be available even without ext_messages.